internal/generator/handlers.go

```diff
@@ -412,19 +521,19 @@ func parseComment(comment *proto.Comment) (string, []map[string]interface{}, []m
 	respExamples := []map[string]interface{}{}
 	message := ""
 	for _, line := range comment.Lines {
-		line = strings.TrimLeft(line, " ")
+		line = strings.TrimLeft(line, " \t")
```

Comment:
can we have just space? we can use `\s` to cover both cases if it's needed

Reply:
The proper way of doing this if you want o make sure to catch unicode char that represents white spaces is to use the following

```suggestion
 	line = strings.TrimLeftFunc(line, unicode.IsSpace))
```

Reply:
`\s` does not work on this line, since it's not regexp.

I'm trying to trim every type of space before the comment line, so that all of these lines would end up with the same result:

```
//foobar
// foobar
//		foobar
//		  foobar
```